src/help/components/entry/Entry.tsx

```diff
@@ -71,9 +72,45 @@ const cleanTextOptions: IOptions = {
     ],
   },
   transformTags: {
-    ...getTransformTags('h1'),
+    ...getTransformTags(headingLevel),
     a: aTransformer,
   },
+});
+
+export const HelpEntryContent = ({
+  data,
+  handleClick,
```

Comment:
Why is this exposed as a prop? I'm unsure what you want to do with it, I would have expected the whole logic to be encapsulated in this component

Reply:
Still not sure what to do with that so I've ended up overriding this change in a further PR